### ISSUE TITLE
logstash 1.5 and oracle java 1.7 don't play well together for plugins

### ISSUE THREAD

**Issue by mitchcapper** [elastic/logstash#3351]
Sadly it seems github has changed their key exchange and it breaks support in oracle java 1.7 due to "Could not generate DH keypair"  when you try to install a new plugin.    It seems this is fixed in 1.8 or in openjdk but not java 1.7.   There should probably just be a notice to avoid oracle 1.7 for now.

**Comment by purbon** [comment]
related to #3354. With build exactly of java7 are you using? as I could check with my own system, using

```
╰─$ java -version
java version "1.7.0_71"
Java(TM) SE Runtime Environment (build 1.7.0_71-b14)
Java HotSpot(TM) 64-Bit Server VM (build 24.71-b01, mixed mode)
``` 
works without this issue.

**Comment by suyograo** [comment]
Closed in favor of #3354
